### PR TITLE
Update bootstrap to version 5.2.3, use package manager to get it

### PR DESCRIPTION
**Pull request by maxnoe**
### Checklist

- [X] All included images are public domain images or their respective licenses are respected (e.g. attribution for CC-BY)
- [X] Images are reduced to around 2000px at the long edge and in jpg format 

The warnings @chrbeckm fixed where actually in code we did not write ourselves, but in the bootstrap scss files, instead of fixing by hand, we should just update.

### CODE REVIEW

**Comment by maxnoe** [comment]
Note: we should switch to installing bootstrap via bundler, not copy the full code into the repo

File: _layouts/carousel-center.html [hunk 1]
```diff
@@ -16,47 +16,47 @@ <h2>
...
-        <span class="sr-only">Previous</span>
-      </a>
-      <a class="carousel-control-next mx-3" style="margin-top: 20vw; align-items: flex-start;" href="#{{ idname }}" role="button" data-slide="next">
-        <span class="carousel-control-next-icon" aria-hidden="true"></span>
-        <span class="sr-only">Next</span>
-      </a>
+        </div>
+      {% endfor %}
     </div>
+    <button class="carousel-control-prev"  href="#{{ idname }}" role="button" data-bs-slide="prev">
```
Comment (The-Ludwig) [inline _layouts/carousel-center.html:51]:
```suggestion
    <button class="carousel-control-prev"  href="#{{ idname }}" role="button" data-bs-slide="prev" data-bs-target="#{{ idname }}">
```

File: _layouts/carousel-center.html [hunk 2]
```diff
@@ -16,47 +16,47 @@ <h2>
...
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </button>
+    <button class="carousel-control-next"  href="#{{ idname }}" role="button" data-bs-slide="next">
```
Comment (The-Ludwig) [inline _layouts/carousel-center.html:55]:
```suggestion
    <button class="carousel-control-next"  href="#{{ idname }}" role="button" data-bs-slide="next" data-bs-target="#{{ idname }}>
```

Comment (The-Ludwig) [reply _layouts/carousel-center.html:55]: Buttons are not dark with the dark theme for some reason, unsure if this fixes it...

**Review by The-Ludwig — CHANGES_REQUESTED** [review]
Nice, almost everything works, only the fading of the bg need an if clause and the carousel needs the dark theme,
see suggestions

File: _layouts/carousel-center.html (hunk 2 above)
Comment (maxnoe) [reply _layouts/carousel-center.html:55]: Dark theme should mean light buttons, right?

**Comment by maxnoe** [comment]
@The-Ludwig Thanks, should be fine now

**Review by The-Ludwig — APPROVED** [review]
good, nice

**Pull request merged**